issue found on: storage/storage.go:358:3: emptyFallthrough

an issue has been detected in source code: ./storage/storage.go:358:3: emptyFallthrough: replace empty case containing only fallthrough with expression list

--- a/storage/storage.go
+++ b/storage/storage.go
@@ -353,9 +353,7 @@ func (storage MemoryStorage) ReadReportForOrganizationAndCluster(
 	switch orgID {
 	case 11940171:
 		return types.ClusterReport(report), errors.New("You have no permissions to get or change info about this organization")
-	case 1, 2, 3:
-		fallthrough
-	case 11789772:
+	case 1, 2, 3, 11789772:
 		report = getReportForCluster(clusterName)
 	}
 

